static/js/notifications.js

```diff
@@ -99,13 +101,10 @@ export function initialize() {
 
 export function update_notification_sound_source(container_elem, settings_object) {
     const notification_sound = settings_object.notification_sound;
-    const audio_file_without_extension = "/static/audio/notification_sounds/" + notification_sound;
-    container_elem
-        .find(".notification-sound-source-ogg")
-        .attr("src", `${audio_file_without_extension}.ogg`);
-    container_elem
-        .find(".notification-sound-source-mp3")
-        .attr("src", `${audio_file_without_extension}.mp3`);
+    const ogg_file = require("../audio/notification_sounds/" + notification_sound + ".ogg");
+    const mp3_file = require("../audio/notification_sounds/" + notification_sound + ".mp3");
+    container_elem.find(".notification-sound-source-ogg").attr("src", ogg_file);
+    container_elem.find(".notification-sound-source-mp3").attr("src", mp3_file);
```

Comment:
Just to check, do we still serve the sound files at the original unhashes path? That is probably an important thing for us to do in order for mobile apps to be able to select that sound option in the future ... but currently the mobile apps just have a fixed sound, so if we stopped having the unhashed URLs, this would not be a current bug so much as an API change that might be annoying in the future.

Reply:
ManifestStaticFilesStorage always keeps the unhashed paths as well.  Unless we write code to remove those explicitly, we'll always have those available.

And all this new code does is pull in a copy of the mp3 and ogg files into `webpack-bundles/` -- the copy in `audio/notification_sounds/` is left as-is and `./tools/update-prod-static` copies that into `~/prod-static/`.